Auctions: Make domain table stackable for mobile users

karlerikounapuu opened the pull request:
Makes the domain table stackable for mobile users on `/auctions` and `/auctions/search`. 

![image](https://user-images.githubusercontent.com/9388771/66393869-546cbc80-e9dc-11e9-8576-70ff54a6152a.png)

Closes #354.

artur-beljajev commented on app/views/auctions/search.html.erb:
```diff
@@ -26,8 +26,20 @@
-                <table class="ui table selectable unstackable fixed" id="auctions-table">
-                    <thead>
+                <table class="ui table selectable mobile stackable fixed" id="auctions-table">
+                    <thead class="mobile-show">
```
Can we use semantic class name and move logic (I assume it is visible only in mobile, based on the name) to CSS?

karlerikounapuu replied:
Renamed to "mobile show", semantic enough I believe. Wouldn't really define deeper selector scope, because it could be heavily referenced from other views as well.

karlerikounapuu commented:
> 1. Any chance to avoid using `!important`?

Possible to avoid `!important` tag in mobile-show class. Removed with last commit. Not that possible with other one since Semantic UI is pretty aggressive when it comes to pushing grid rules.

> 2. Why `scope="col"` is needed?

Came from habit, to be honest. Removed with last commit.

artur-beljajev commented:
@karlerikounapuu I wonder if app-specific overrides should really reside in `app/javascript/src/semantic-ui/`. I can't see many changes here https://github.com/internetee/auction_center/commits/master/app/javascript/src/semantic-ui.

Perhaps we should put it here `app/javascript/src/common` or here `app/javascript/src/global/`?

karlerikounapuu commented:
> @karlerikounapuu I wonder if app-specific overrides should really reside in `app/javascript/src/semantic-ui/`. I can't see many changes here https://github.com/internetee/auction_center/commits/master/app/javascript/src/semantic-ui.
> 
> Perhaps we should put it here `app/javascript/src/common` or here `app/javascript/src/global/`?
> 
> P.S. We outsource front-end part. It should be easy to integrate it.

@artur-beljajev Changing file path for overrides makes sense. I've moved them to `app/javascript/src/global/`

artur-beljajev commented on app/javascript/src/global/_common.scss:
```diff
@@ -158,4 +158,22 @@ strong {
+        }
+    }
+
+    #auctions-table > thead > tr {
```
Such nesting is quite brittle. Might be better to use classes.

artur-beljajev commented on app/javascript/src/semantic-ui/definitions/collections/grid.less:
```diff
@@ -1917,4 +1917,4 @@
 
 
-.loadUIOverrides();
+.loadUIOverrides();
```
What is changed here?

artur-beljajev commented on app/javascript/src/semantic-ui/definitions/collections/table.less:
```diff
@@ -1144,4 +1144,4 @@
 }
 
-.loadUIOverrides();
+.loadUIOverrides();
```
What is changed here?
